tutorials/3d/portals/appendix.rst

```diff
@@ -0,0 +1,87 @@
+
+.. image:: img/room_points.png
+
+Manually editing rooms points is more difficult than editing portal points, but it can be useful in some situations, especially where the auto-bound doesn't _quite_ get the right result you want. It is usually a good idea to use a lot of ``simplification`` in the inspector for the Room before generating the points (be aware though that by default, the ``simplification`` value will be inherited from the RoomManager).
+
+RoomManager
+~~~~~~~~~~~
+
+Show Debug
+^^^^^^^^^^
+This can be used to turn on and off display of portals in the editor, and control the amount of logging. Debug will always be set to false on exported projects.
+
+Debug Sprawl
+^^^^^^^^^^^^
+This mode will only display meshes that are sprawling through portals from the current camera room. Large statics that cross portals are usually the ones you want to sprawl. Typical examples might be terrain mesh areas, or large floor or ceiling meshes. You usually don't want things like door frames to sprawl to the adjacent room - that is what fine tuning the ``Portal Margin`` is for.
+
+Merge Meshes
+^^^^^^^^^^^^
+In order to keep drawcalls to a minimum, the system offers the option to automatically merge similar ``STATIC`` meshes within a room (aka, *static batching*). This can increase performance in many cases. The culling accuracy is reduced, but as a room is a fairly logical unit for culling, this trade off usually works in your favour.
+
+Plane Simplification
+^^^^^^^^^^^^^^^^^^^^
+In some cases, automatically generated convex hull bounds may contain a very large number of planes. This is not ideal because it slows down determining which room a camera or object is within. The system can optionally simplify hulls. The degree of simplification can be selected by the user, between ``0`` (no simplification) and ``1`` (maximum simplification). You can also override this value in individual rooms.
+
+Portals
+~~~~~~~
+
+Portal Active
+^^^^^^^^^^^^^
+Portals can be turned on and off at runtime. This is especially useful if you have open and closing doors.
+
+Two Way
+^^^^^^^
+Portals can either be two way or one way. One way portals may be useful for example to create windows that can be seen out of, but not seen into. This can help performance when viewing buildings from outdoors.
+
+Particle Systems
+~~~~~~~~~~~~~~~~
+Be aware that when placing ``STATIC`` particle systems, the ``AABB`` on conversion may have zero size. This means the particle system may be unexpectedly culled early. To prevent this, either set the particle system ``portal mode`` to ``DYNAMIC``, or alternatively, add an ``extra cull margin`` to the particle system in the Geometry Inspector.
```

Comment:
```suggestion
Particle Systems
~~~~~~~~~~~~~~~~

Be aware that when placing ``STATIC`` particle systems, the AABB on conversion may have a size equal to zero. This means the particle system may be unexpectedly culled early. To prevent this, either set the particle system ``portal mode`` to ``DYNAMIC``, or alternatively, define an **Extra Cull Margin** to the particle system in the particle node's GeometryInstance3D section in the inspector.
```